source/uwp/Renderer/dll/AdaptiveCards.Rendering.Uwp.def

```diff
@@ -1,6 +1,5 @@
 LIBRARY "AdaptiveCards.Rendering.Uwp.dll"
 
 EXPORTS
-    DllGetClassObject           PRIVATE
```

Comment:
@jonwis any idea why it was exported before?
and what does it exactly do? what does **private** mean here?
do we need to export **DllMain**?
is this equivalent of **__declspec(dllexport)**?

Reply:
This function is needed if the DLL exposes COM objects (CoCreateInstance).

You do not export DllMain. Its address is recorded in the module header as the DLL entry point, to be called by the loader. Nobody needs to GetProcAddress for DllMain.

"PRIVATE" means "Export the function by name but do not include it in the export library."  That way, people who link to `AdaptiveCards.Rendering.Uwp.lib` do not get multiple definitions of `DllGetClassObject`.